src/main/scala/uncore/tilelink2/Parameters.scala

```diff
@@ -56,7 +56,7 @@ case class TLManagerParameters(
 
   // The device had better not support a transfer larger than it's alignment
   val minAlignment = address.map(_.alignment).min
-  require (minAlignment >= maxTransfer)
+  require (minAlignment >= maxTransfer, "minAlignment (" + minAlignment + ") must be >= maxTransfer (" + maxTransfer + ")")
```

Comment:
For future ref, these kind of strings can be less annoyingly written with string interpolation, by writing an ```s``` before the open-quotes, as in

    require (minAlignment >= maxTransfer, s"minAlignment ($minAlignment) must be >= maxTransfer ($maxTransfer)")